tests/unit/confidant/authnz/authnz_test.py

```diff
@@ -3,6 +3,11 @@
 from mock import Mock
 from werkzeug.exceptions import Unauthorized
 
+# Prevent call to KMS when app is imported
+from confidant import settings
+settings.encrypted_settings.secret_string = {}
```

Comment:
This is ok for now, but it's likely better to patch these directly in the used functions, so there's not unintended side effects across tests.